.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-2eb4a60ed14a38260b85b0c765161f0ce45be6d1
+fc5e0f7734ec599954b15e29236903f09867307f
```

Comment:
intentional?

Reply:
XLA needs update because it overrides the interface.